backend/src/chats/chat/chat.controller.ts

```diff
@@ -13,6 +13,7 @@ import { CreateChatDto } from './dto/create-chat.dto';
 import { UpdateChatDto } from './dto/update-chat.dto';
 import { Chat } from './entities/chat.entity';
 import * as bcrypt from 'bcrypt';
+import {} from 'typeorm';
```

Comment:
I'm curious what this does?

Reply:
vscode complained that it could not infer return type of create/update methods without a reference to typeorm.... 🤷